docs/src/_concepts/state_backends.md

```diff
@@ -5,56 +5,85 @@ layout: doc
 weight: 3
 ---
 
-Many Meltano pipelines load data incrementally.
-In order for these pipelines to run efficiently and properly without dropping historical data or loading duplicate records, Meltano needs to keep track of the state of a pipeline as part of each run.
+Meltano is capable of running pipelines that load data incrementally.
+In order for these pipelines to run efficiently and properly without losing historical data or loading duplicate records, Meltano needs to keep track of the state of a pipeline as part of each run.
 This state can be stored in Meltano's backend system database, but
 for Meltano projects running in ephemeral environments or in circumstances where administering a dedicated backend database is undesirable, Meltano also supports persisting state in remote state backends backed by cloud storage services.
 
+## Supported Backends
+
+* [System Database](#default-system-databse)
+* [Local Filesystem](#local-filesystem)
+* [Amazon AWS S3](#aws-s3)
+* [Azure Blob Storage](#azure-blob-storage)
+* [Google Cloud Storage](#google-cloud-storage)
+
 ## Installation
 
-To cut down on extraneous dependencies, Meltano imports cloud storage client libraries lazily and these libraries are not included in the base `meltano` package.
+No extra work is needed to use the default system database or local filesystem as a state backend as they are already part of any base Meltano distribution.
 
-Only the default system database state backend _or_ the local filesystem state backend can be used as part of the base `meltano` distribution.
+To use a cloud storage backend, install Meltano using one of the following [extras](https://peps.python.org/pep-0508/#extras):
 
-To use a cloud storage backend, install Meltano using one of the following (extras)[https://peps.python.org/pep-0508/#extras]:
 - `meltano[s3]` to use the AWS S3 state backend.
 - `meltano[azure]` to use the Azure Blob Storage state backend.
 - `meltano[gcs]` to use the Google Cloud Storage state backend.
 
```

Comment:
Do we have guidance on what to do if a user has already installed Meltano? Is it just a reinstall? Should they do something else?

Reply:
Good callout. The user would just need to install the relevant extra (e.g. `pip install "meltano[s3]"` and pip will only install the missing dependencies.

I also added a callout that the user will need to `meltano upgrade` to 2.10+ in order to use state backends.